libipl/p10/ipl0.C

```diff
@@ -537,7 +537,7 @@ static int ipl_set_ref_clock(void)
 
 				ipl_log(IPL_ERROR, "Attribute CP_REFCLOCK_SELECT update failed"
 					" for proc %d \n", pdbg_target_index(proc));
-				ipl_error_callback(IPL_ERR_ATTR_ACCESS);
+				ipl_plat_procedure_error_handler(IPL_ERR_ATTR_ACCESS);
```

Comment:
I don't have a strong opinion on this, @ojayanth 
If the attribute is not present, we can say a software error but, attribute access might happen for software corruption also and even that happens platform need to restart the application (like BMC doing for start host service)

Reply:
IPL failure triggers restarting the ipl sepecific service.  starthost service in is short living service.

Reply:
@rajez133 I don't have a strong opinion on this, you can take a call based on this use case.

Reply:
I talked to jayanth. He agrees to keep procedure callout for this error.